lib/application/coordinator/routes.dart

```diff
@@ -87,3 +87,16 @@ class SettingsPath extends AppPath {
   @override
   String get formattedPath => '/$name';
 }
+
+//
+// Update Collection
+//
+class UpdateCollectionPath extends AppPath {
+  UpdateCollectionPath({this.collectionId});
+  static const name = 'update_collection';
+
+  final String? collectionId;
```

Comment:
Don't we already have all the collection metadata once we get to this route? Shouldn't we simply fetch the associated `collection/$id/memos` once we are actually in the route?

Reply:
We have no use-case that moves entities between routes. Do you think that we should introduce `transients` here? I mean, this route would receive an optional `transient` entity named `CollectionMetada`

Reply:
That sure is a good question, I'm not sure. This is the same problem that we are facing in multiple other projects, which is: how do we handle "passing data" around multiple pages/states. I don't have a good enough answer now, so I think we could move on with the current solution.

Reply:
By current solution you mean using just the id or using the transient?